examples/app-pg/main.py

```diff
@@ -3,25 +3,30 @@
 import os
 import time
 import pyodbc
+from faker import Faker
+
+fake = Faker()
 
 CONNECTION_STRING = 'DRIVER={{PostgreSQL Unicode}};SERVER={server};DATABASE={database};UID={username};PWD={password};'
 
 
 SQL_CREATE_TABLE = '''
-    CREATE TABLE fruits (
+    CREATE TABLE users (
         id INT,
         name VARCHAR(50),
-        quantity INT
+        city VARCHAR(50)
     )
 '''
 
-SQL_INSERT_DATA = 'INSERT INTO fruits (id, name, quantity) VALUES (?, ?, ?)'
+SQL_INSERT_DATA = 'INSERT INTO users (id, name, quantity) VALUES (?, ?, ?)'
+
+DATA = []
```

Comment:
```suggestion
data = []
```
Previously DATA was all caps because we were using the 'data' as constant. Now, since you are using 'data' as a variable. Change it to lowercase as well.